docs: added global flags information to the doc (#458)
kurtosis-tech/kurtosis: docs/docs/cli-reference/index.md

kurtosis version
```

### Global Flags

> Kurtosis CLI supports two global flags - `help` and `cli-log-level`. These flags can be used with any Kurtosis CLI commands.

Reviewer: ```suggestion
The Kurtosis CLI supports two global flags - `help` and `cli-log-level`. These flags can be used with any Kurtosis CLI command.
```

kurtosis version
```

### Global Flags

> Kurtosis cli supports two global flags - `help` and `cli-log-level`. These flags can be used with any kurtosis cli commands.

Reviewer: ```suggestion
Kurtosis CLI supports two global flags - `help` and `cli-log-level`. These flags can be used with any Kurtosis CLI commands.
```

#### -h or --help

> This flag prints the helptext for all commands and subcommands. You can use this at any time to see information on the command you're trying to run. For example:

Reviewer: ```suggestion
This flag prints the help text for all commands and subcommands. You can use this at any time to see information on the command you're trying to run. For example:
```

```

> kurtosis service -h

Reviewer: Can we give the user an example of the expected output?

```

#### cli-log-level

> This flag sets the level that kurtosis cli will log at - by default it only logs `info` level to the cli. Thesea are all the levels supported by kurtosis -

Reviewer: ```suggestion
This flag sets the level of details that the Kurtosis CLI will print logs with - by default it only logs `info` level logs to the CLI. The following other log levels are supported by Kurtosis -
```

```panic|fatal|error|warning|info|debug|trace```. For example, logs with error level can be printed using the command below:- 

```

> kurtosis run --cli-level-log error github.com/package-author/package-repo 

Reviewer: Can you add an example output?

```

:::info

> Users can use the `--cli-log-level` flag to display entire flag traces to the cli. By default the entire stack-traces are logged on to `kurtosis-cli.log` file. The command below, for example, will display the entire stack-traces to the cli for debugging purpose.

Reviewer: ```suggestion
Users can use the `--cli-log-level` flag to display the entire stack trace to the CLI. By default the entire stack trace is saved to the `kurtosis-cli.log` file. The command below, for example, will display the entire stack-traces to the CLI for debugging purposes:
```

```

> kurtosis run --cli-log-level debug github.com/package-author/package-repo 

Reviewer: Would love an example output here too.
